### ISSUE TITLE
FP on rule multiple-variable-declarations-same-scope within mixin calls when parameters are the same variables

### ISSUE THREAD

**Issue by ehats** [racodond/sonar-css-plugin#75]
Hello!

The following piece of code seems to be causing an unnecessary violation from this plugin:

```
@myvar: 1px;

.my-mixin (a, b) {
 margin: @a @b;
}

.my-class {
 .my-mixin(@myvar, @myvar);
}
```

I'm getting a `Merge those multiple variable declarations within the same scope` violation on the line of the mixin call (`.my-mixin(@myvar, @myvar)`). It seems that the plugin detects the mixin parameters as variable declarations.

Let me know if you need any further details!

SonarQube version: 6.2
sonar-css-plugin version: 4.11-SNAPSHOT (none of the newer changes seem related to this issue though)

**Comment by racodond** [comment]
Hi @ehats,

Thanks for your feedback!
You're right, there's an issue. I'll fix it as soon as I can.

David
